DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/SendBinding.cs

```diff
@@ -55,6 +55,15 @@ public List<ISendFilter> GetSendFilters()
 
   public async Task Send(string modelCardId)
   {
+    // try
+    // {
+    //   throw new SpeckleException("Something bad happened");
+    // }
+    // catch (SpeckleException ex)
+    // {
+    //   throw new AggregateException("Wrapping this one up", ex);
+    // }
```

Comment:
I'm assuming this was added by mistake 😅

Reply:
hehe, I thought I'd removed this stuff. I was debugging a crash that only occurred outside of the IDE